charts/zenko/requirements.yaml

```diff
@@ -14,14 +9,21 @@ dependencies:
 - name: redis
   version: "1.1.12"
   repository: "https://kubernetes-charts.storage.googleapis.com/"
+- name: kafka
+  version: "0.7.1"
```

Comment:
Can we upgrade Kafka to 1.0.x?

Reply:
This is the Chart version, not the `appVersion`: https://github.com/kubernetes/charts/blob/753273e6d2e06f656e7eb8730fbbda6da13a495f/incubator/kafka/Chart.yaml#L6

Reply:
Exactly, as Nicolas said, this is the chart version but I believe the Kafka version used in the Confluent 4.0 docker image used in this chart is Kafka 1.0